cmd/spire-server/cli/federation/create.go

```diff
@@ -0,0 +1,256 @@
+package federation
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"flag"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/mitchellh/cli"
+	"github.com/spiffe/go-spiffe/v2/spiffeid"
+	trustdomainv1 "github.com/spiffe/spire-api-sdk/proto/spire/api/server/trustdomain/v1"
+	"github.com/spiffe/spire-api-sdk/proto/spire/api/types"
+	"github.com/spiffe/spire/cmd/spire-server/util"
+	common_cli "github.com/spiffe/spire/pkg/common/cli"
+	"google.golang.org/grpc/codes"
+)
+
+const (
+	profileWeb    = "web"
+	profileSPIFFE = "spiffe"
+)
+
+// NewCreateCommand creates a new "create" subcommand for "federation" command.
+func NewCreateCommand() cli.Command {
+	return newCreateCommand(common_cli.DefaultEnv)
+}
+
+func newCreateCommand(env *common_cli.Env) cli.Command {
+	return util.AdaptCommand(env, new(createCommand))
+}
+
+type createCommand struct {
+	path string
+	fr   FederationRelationship
+}
+
+// FederationRelationship type is used for parsing federation relationships from file
+type FederationRelationship struct {
+	TrustDomain           string `json:"trust_domain"`
+	BundleEndpointURL     string `json:"bundle_endpoint_url"`
+	BundleEndpointProfile string `json:"bundle_endpoint_profile"`
+	EndpointSPIFFEID      string `json:"endpoint_spiffe_id"`
+	BundlePath            string `json:"bundle_path"`
```

Comment:
Hmm, I wonder if this experience (i.e. loading from file) might be better if the bundle is embedded here. We could use `json.RawMessage` and then use bundle format to either parse out a `spiffebundle.Bundle` or certificate PEM...

Thoughts?